Loadouts/blu_usmc_m16_desert.hpp

```diff
@@ -0,0 +1,260 @@
+//Author: BWMF
```

Comment:
@PabstMirror we've logged #39 specifically to avoid relying on RHS units (and any issues they may cause). I think this might be a step back from that.